### DISCUSSION TITLE
New to Next js - Need to call external api

### DISCUSSION THREAD

**Discussion by dograsahil90** [vercel/next.js#64800] [Help]
### Summary

I am building a front end app using Next js and would like to call external back end api(Spring api). The confusion I have .. as we know Next js have both Client and server side functionality but in my case the server side is handled by the external backend api.

So do i have to still use Next js server side(/api  or /Route Handlers or Server Actions) to call the external api or there are other better options.

The concern I have is adding another server side layer will impact the overall performance ?

**Comment by vaibhav-OP** [comment]
Running your server separately will not impact any overall performance problems. 
In both the cases your server will be hosted somewhere and the frontend will be calling it.
